### PR TITLE
Improvements, fixes #24

### LINKED ISSUE #24: README is outdated
```
$ mkdocs service
Usage: mkdocs [OPTIONS] COMMAND [ARGS]...

Error: No such command "service".
$ mkdocs serve
INFO    -  Building documentation...

Config file 'syselevenstack-documentation/mkdocs.yml' does not exist.
$ cd english/
$ mkdocs serve
INFO    -  Building documentation...
ERROR   -  Config value: 'theme_dir'. Error: The path /usr/share/mkdocs/themes/mkdocs_bootswatch/flatly/ doesn't exist

Aborted with 1 Configuration Errors!
```

So how to actually test the documentation? @LarsFronius maybe you know it?

### PR DESCRIPTION
**Pull request by dschwabeS11**
Please review, don't merge yet, English translation follows.
Thanks!

### CODE REVIEW

**Comment by dschwabeS11** [comment]
@stefreak Please review the English tutorial as well. Ty

File: german/docs/tutorials/affinity.md [hunk 1]
```diff
@@ -197,7 +193,7 @@ openstack server show server_1 -c name -c hostId
 * [Nova Scheduler Affinity Filter](http://docs.openstack.org/mitaka/config-reference/compute/scheduler.html#servergroupaffinityfilter)
 * [OpenStack Client Server Create](http://docs.openstack.org/developer/python-openstackclient/command-objects/server.html#server-create)
 * [OpenStack Client ServerGroup](http://docs.openstack.org/developer/python-openstackclient/command-objects/server-group.html)
-
+*
 ## Weiterführende Links/Beispiele
 
-Das auf Github veröffentlichte Beispiel [affinity](https://github.com/syseleven/heattemplates-examples/tree/master/affinity) enthält ein Beispiel für affinity und anti-affinity.
\ No newline at end of file
+Das auf Github veröffentlichte Beispiel [affinity](https://github.com/syseleven/heattemplates-examples/tree/master/affinity) enthält ein Beispiel für *affinity* und *anti-affinit*.
```
Comment (dschwabeS11) [inline german/docs/tutorials/affinity.md]: Fixed

File: english/docs/tutorials/firststeps.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* In order to import your SSH-key using the dashboard, we go to "Compute" and "Access And Security" and from there to "Key Pairs".
+* There we select "Import Key Pair", give the key pair a name (that we remember for later use) and import the public part of the key pair via copy & paste into the interface. 
+
+Once submitted we can use the key pair with this name in the API and Templates.
+
+![import SSH key](../img/sshkeys.png)
+
+## Starting the compute instance
+
+* We start our template at *Orchestration --> Stacks --> Launch Stack*.
```
Comment (dschwabeS11) [inline english/docs/tutorials/firststeps.md]: Changed in English and German firststeps.md

File: german/docs/tutorials/openstack-cli.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+**PIP**
+
+Damit wir die benötigten Pakete installieren können, nutzen wir das Paketverwaltungsprogramm [PIP](https://de.wikipedia.org/wiki/Pip_(Python)).
+
+`PIP` installieren wir mit folgendem Befehl:
+```shell
+easy_install pip
+```
+
+**Virtualenv**
```
Comment (dschwabeS11) [inline german/docs/tutorials/openstack-cli.md]: Done as discussed.

File: english/docs/tutorials/openstack-cli.md [hunk 4]
```diff
@@ -20,66 +20,46 @@ This tutorial serves as manual on how to install the OpenStack CLI (Command Line
...
 ```
 
-If we want `virtualen`to inherit global installed packages (e.g IPython or Numpy) we have to use the following command:
-```shell
-virtualenv venv --system-site-packages
-```
+**Python dependencies**
 
-This commands create a virtual subfolder within the projectfolder into which everything is beeing installed. However we have to active active the folder first (within the terminal that we use to work on the project):
+To work arround an OSX issue run the following command:
```
Comment (stefreak) [inline english/docs/tutorials/openstack-cli.md]: *around* (1 r zu viel)

**Pull request merged**